src/components/Image/Image.ts

```diff
@@ -116,8 +116,8 @@ function validateProps(element: Image) {
     throw new Error(`Invalid layout: ${element.layout}. Allowed values are 'fixed', 'constrained', 'fullWidth'.`)
   }
   if (element.layout !== "fullWidth") {
```

Comment:
Also consider checking if element.layout is not undefined? Because, we need the check only when layout value is supplied and is not fullWidth. This would allow usages like 
```
<nosto-image src="..." width="320"></nosto-image>
<nosto-image src="..." height="240"></nosto-image>
```

Reply:
there is a default for layout being applied

Reply:
the implied default for layout is "constrained"